purchase_manual_delivery/models/purchase_order.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2019 ForgeFlow S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    pending_to_receive = fields.Boolean(compute="_compute_pending_to_receive")
+    manual_delivery = fields.Boolean(
+        string="Manual delivery",
+        compute="_compute_manual_delivery",
+        help=(
+            "Stock transfers need to be created manually to receive this PO's products"
+        ),
+        readonly=False,
+        store=True,
+    )
+
+    @api.depends("company_id")
+    def _compute_manual_delivery(self):
+        """The manual delivery option is derived from the company of the order"""
+        for po in self:
+            po.manual_delivery = po.company_id.purchase_manual_delivery
+
+    def _compute_pending_to_receive(self):
+        for order in self:
+            order.pending_to_receive = True
+            if not any(order.mapped("order_line.pending_to_receive")):
+                order.pending_to_receive = False
```

Comment:
`order.pending_to_receive = any(order.mapped("order_line.pending_to_receive")` would be simpler? Also I don't think you want orders with no line to be pending_to_receive.

Reply:
Agreed!